app/services/usps_in_person_proofing/transliterator.rb

```diff
@@ -0,0 +1,46 @@
+module UspsInPersonProofing
+  class Transliterator
+    # This is the default. May not be able to override this in current version.
+    REPLACEMENT = '?'.freeze
+
+    # Container to hold the results of transliteration
+    TransliterationResult = Struct.new(
+      # Was the value different after transliteration?
+      :changed?,
+      # Original value submtted for transliteration
+      :original,
+      # Transliterated value
+      :transliterated,
+      # Characters from the original that could not be transliterated
+      :unsupported_chars,
+    )
```

Comment:
I believe we need `keyword_init: true` for the constructor to act the way we want?
```suggestion
      keyword_init: true,
    )
```

Reply:
Somehow the testing showed this working, but... looks like it should have this, so added.